index.bs

```diff
@@ -126,16 +126,23 @@ A [=crash report=]'s [=report/body=], represented in JavaScript by
   It represents whether the visual viewport that the crashed {{Document}} was hosted in was
   visible at all, or fully occluded.
 
-Note: Crash reports are always delivered to the [=endpoint=] named `default`;
-there is currently no way to override this.  If you want to receive other kinds
-of reports, but not crash reports, make sure to use a different name for the
-endpoint that you choose for those reports.
-
 Note: Crash reports are not observable to JavaScript, as the page which would
 receive them is, by definition, not able to. The IDL description of
 CrashReportBody exists in this spec to provide a JSON-seriaizable interface
 whose serialization can be embedded in the out-of-band reports.
 
+Override the [=endpoint=] {#override-endpoint}
+-------------------------
+By default, crash reports are delivered to the [=endpoint=] named `default`,
+which is also upload other kinds of reports. To receive only crash reports,
+you can specify the endpoint named `crash-reporting`.
```

Comment:
I think this paragraph could use some better wording. Maybe something like:

```
By default, crash reports are delivered to the `default` [=endpoint=],
which receives many other kinds of reports besides crash reports.
Developers can supply a separate URL to the well-known [=endpoint=]
named `crash-reporting`, to direct crash reports there, instead of the
`default` [=endpoint=].
```